samples/nrf9160/modem_trace_flash/Kconfig.trace_flash

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+module = MODEM_TRACE_FLASH_BACKEND
+module-str = Modem trace flash backend
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+if NRF_MODEM_LIB_TRACE
+
+# Adds NRF_MODEM_LIB_TRACE_BACKEND_CUSTOM to the trace backend choice.
+choice NRF_MODEM_LIB_TRACE_BACKEND
+
+config NRF_MODEM_LIB_TRACE_BACKEND_FLASH
```

Comment:
Can we just remove this new KConfig option? I believe the intention was to have this if this feature was in a library. But since this is now only in the sample, we might as well remove this

Reply:
nevermind, I see we need it to get the partition manager to allocate a partition in the external flash for traces.